src/components/export-modal/export-modal.js

```diff
@@ -9,8 +9,8 @@ import './export-modal.scss';
 /**
  * Modal to allow users to choose between SVG/PNG export formats
  */
-const ExportModal = ({ graphSize, theme, onToggle, visible }) => {
-  if (!visible.exportBtn) {
+const ExportModal = ({ graphSize, theme, onToggle, visible, display }) => {
+  if (!display.exportBtn) {
```

Comment:
[Nit] This can be ignored, but I would like to add a suggestion here which might need some refactor. I feel this check needs to be done before calling ExportModal component (i.e., the check needs to be present in the parent component where ExportModal is called). The modal should only be called if it passes the condition.